avaframe/tests/test_geoTrans.py

```diff
@@ -460,3 +460,72 @@ def getIPZ(z0, xEnd, yEnd, dx):
     zv = z0 - np.tan(np.radians(meanAlpha)) * x
 
     return zv
+
+
+def test_cartToSpherical():
+    """ test converting to spherical coordinates """
+
+    # setup required inputs
+    X = 10.
+    Y = 10.
+    Z = np.sqrt(10.**2 + 10.**2)
+
+    # call function to be tested
+    r, phi, theta = geoTrans.cartToSpherical(X, Y, Z)
+
+    assert r == np.sqrt(10.**2 + 10.**2 + Z**2)
+    assert phi == 45.
+    assert theta == 45.
+
```

Comment:
same here: could we try to use other angles as 45? 45 is a very bad choice in my opinion

Reply:
there is a second test case just the lines below where it is not 45, just kept both